src/components/Dialogs/GeneralDialog/GeneralDialog.style.ts

```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled'
+import { css } from '@emotion/react'
+import { Button } from '@/shared/components'
+import { colors, sizes, breakpoints, zIndex } from '@/shared/theme'
+
+export const StyledBackdrop = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.4);
+  z-index: ${zIndex.globalOverlay};
+  @media (min-width: ${breakpoints.medium}) {
+    padding-left: var(--scrollbar-gap-width);
+  }
+`
+
+export const StyledContainer = styled.div`
+  position: relative;
+  width: 90%;
+  max-width: ${sizes(110)};
+  min-height: 150px;
+  margin: ${sizes(32)} auto;
+  background-color: ${colors.gray[600]};
+  padding: ${sizes(4)};
+  box-shadow: 0px 8px 8px rgba(0, 0, 0, 0.12), 0px 24px 40px rgba(0, 0, 0, 0.16);
+
+  @media screen and (min-width: ${breakpoints.small}) {
+    padding: ${sizes(6)};
+  }
+`
+
+export const StyledExitButton = styled(Button)`
```

Comment:
I think this click target may be too big (it's 80x70). We could try reducing the padding on the button a bit